code/script.js

```diff
@@ -244,100 +243,132 @@ const generateBoard = () => {
   })
 }
 
-// Randomly select a person from the characters array and set as the value of the variable called secret
 const setSecret = () => {
   secret = charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)]
+  console.log(secret);
 }
 
-// This function to start (and restart) the game
 const start = () => {
-  // Here we're setting charactersInPlay array to be all the characters to start with
   charactersInPlay = CHARACTERS
-  // What else should happen when we start the game?
+  setSecret();
+  generateBoard();
+  winOrLosePage.classList.toggle("hidden");
 }
 
-// setting the currentQuestion object when you select something in the dropdown
 const selectQuestion = () => {
-  const category = questions.options[questions.selectedIndex].parentNode.label
-  // This variable stores what option group (category) the question belongs to.
-  // We also need a variable that stores the actual value of the question we've selected.
+  const category = questions.options[questions.selectedIndex].parentNode.label;
+  const selectedElementValue = questions.options[questions.selectedIndex].value;
+  const selectedElementInnerText = questions.options[questions.selectedIndex].innerText;
 
   if (category === 'hair color') {
     currentQuestion = {
       attribute: 'hairColor',
-      // value: ,
-      // 👆 add the value from the input here
+      value: selectedElementValue,
+      innerText: selectedElementInnerText,
```

Comment:
It is not a good practice to name fields the same way that DOM functions or fields.